README.md

```diff
@@ -1,109 +1,67 @@
 Overview
 ========
 
-pa11ycrawler is a python command line tool built using [Scrapy](http://doc.scrapy.org/en/latest/index.html) and [Pa11y](http://pa11y.org/) for
-crawling a website and storing a Pa11y report for each page.
-
-The reports can be in the form of any of Pa11y's default report types or [1.0-json](https://github.com/springernature/pa11y-reporter-1.0-json).  There is an additional command to produce html reports from the 1.0-json.
+pa11ycrawler a [Scrapy](http://doc.scrapy.org/en/latest/index.html)
+spider that calls [Pa11y](http://pa11y.org/) on every page of a locally-running
+Open edX installation, to audit it for accessibility purposes.
+It will store the result of each page audit in a data directory as a set of
+JSON files, which can be transformed into a beautiful HTML report.
 
 Installation
 ============
 
-Prereqs
--------
-
-* python (v2.7)
-* node.js
-
-Install node Requirements
--------------------------
-```
-npm install -g pa11y@3.6.0 pa11y-reporter-1.0-json@1.0.2
```

Comment:
right-o; i had missed the global install previously. thanks for cleaning that up.